packages/synapse-react-client/mocks/msw/handlers/accessRequirementHandlers.ts

```diff
@@ -61,6 +61,8 @@ export const getAccessRequirementHandlers = (backendOrigin: string) => [
       return res(ctx.status(status), ctx.json(response))
     },
   ),
+]
+export const getAccessRequirementStatusHandlers = (backendOrigin: string) => [
```

Comment:
Split status handlers into a separate function because MSW in Storybook doesn't let you override handlers that are already declared